testfiles/RadLoHydrHeatCoolAuto.idf

```diff
@@ -1951,19 +1951,19 @@
     0.013,                   !- Hydronic Tubing Inside Diameter {m}
     autosize,                !- Hydronic Tubing Length {m}
     MeanAirTemperature,      !- Temperature Control Type
-    HeatingDesignCapacity,   !- Heating Design Capacity Method
-    Autosize,                !- Heating Design Capacity {W}
+    FractionOfAutosizedHeatingCapacity,   !- Heating Design Capacity Method
+    ,                        !- Heating Design Capacity {W}
     ,                        !- Heating Design Capacity Per Floor Area {W/m2}
-    ,                        !- Fraction of Autosized Heating Design Capacity
+    1.0,                     !- Fraction of Autosized Heating Design Capacity
```

Comment:
This bug fix, was extended to illustrate the different scalable sizing methods in one example file.  At the same time I was trying to avoid diffs.